x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_bulk_actions.test.tsx

```diff
@@ -124,6 +126,55 @@ describe('bulk action hooks', () => {
       expect(openExistingCase).toHaveBeenCalled();
     });
 
+    it('should open the flyout from the case modal', async () => {
```

Comment:
An e2d test for this would also be nice if there is time. Looked a bit for them but dunno if there even are any regarding bulk actions in the alerts table.

Reply:
I don't think we have. Do you mind if we do it on another PR? I have this issue to track it https://github.com/elastic/kibana/issues/160240.